### ISSUE TITLE
Multiple ifDefined() directives don't work properly

### ISSUE THREAD

**Issue by justinfagnani** [Polymer/lit-html#1066]
# Description

Only the last ifDefined() instance is respected. If the last value is defined, the committer is dirtied and setAttribtue() is called, causing the attribute to be rendered with the ToString value of the directive rendered to the attribute.

# Steps to Reproduce

1. Write this code

```ts
html`<div foo="${ifDefined('a')}${ifDefined(undefined)}"></div>">`
```

2. See this output...

😬

**Comment by kevinpschaaf** [comment]
Repro showing the general result of having `noChange` at different positions in a multi-part attribute binding: https://jsbin.com/mivoriz/edit?html,output

**Comment by justinfagnani** [comment]
This is fixed in 2.0
